README.md

```diff
@@ -8,9 +8,13 @@
 - Redux
```

Comment:
[OPTIONAL] It would be good to include a detailed description of your project that explains its purpose to others.